packages/core/src/Badge/Badge.spec.js

```diff
@@ -2,6 +2,13 @@ import React from 'react'
 import { Badge, theme } from '..'
 
 describe('Badge', () => {
+  let consoleError
+  beforeAll(() => {
+    consoleError = console.error
+    console.error = jest.fn()
+  })
+  afterAll(() => (console.error = consoleError))
```

Comment:
afterEach

Reply:
Refactored the `afterAll` hooks to be `afterEach` hooks!